client/components/hoc/protect-form/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import debugModule from 'debug';
+import page from 'page';
+import i18n from 'i18n-calypso';
+import { without } from 'lodash';
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:protect-form' );
+const confirmText = i18n.translate( 'You have unsaved changes. Are you sure you want to leave this page?' );
```

Comment:
yeah! I thought about it, but in that case i'll have to omit those `localize` props (translate, moment ect...) from the props I pass down, that what annoyed be about it. But yeah I'm ok to do that.